### PR TITLE
Loading speed up

### PR DESCRIPTION
**Pull request by moringspeaker**
I found out that our neighborhood page loads too slow due to bs icon which will be loaded from Bootstrap cdn in every iteration. So I stored it in directory: /static/bootsrap_icon/. And I also wrote some dynamically updating DOM JS script just for fun.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -138,3 +138,7 @@ dmypy.json
 .DS_Store
 .vscode/
 .idea/
+.env
+CACHE/
+db.sqlite3
+.gitignore
```
Comment (vchrombie) [inline .gitignore:144]: all these changes are not related and not useful

it would be great if you can remove them

File: neighborhood/views.py [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 from .utils import get_title
-
 from decouple import config
+from django.core import serializers
 
 
 def neighborhoods(request):
     neighborhoods = Neighborhood.objects.all()
-
+    neighborhoods_json = serializers.serialize("json", neighborhoods)
```
Comment (vchrombie) [inline neighborhood/views.py:10]:
```suggestion
    neighborhoods = serializers.serialize("json", neighborhoods)
```

File: neighborhood/views.py [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
     context = {
-        "neighborhoods": neighborhoods,
         "page": "neighborhoods",
         "MAPBOX_API_KEY": config("MAPBOX_API_KEY"),
     }
-
     if request.user.is_authenticated:
         context["firstname"] = request.user.first_name
-
+    context["neighborhoods"] = neighborhoods_json
```
Comment (vchrombie) [inline neighborhood/views.py:17]: I would suggest to revert this change as it doesn't really improve anything. The previous change just works the same way.

Please let me know if I am missing anything.

File: tests/test_views.py [hunk 4]
```diff
@@ -687,8 +688,12 @@ def test_neighborhoods_view(self):
...
         # Check that the rendered context contains the list of neighborhoods.
         self.assertIn("neighborhoods", response.context)
-        neighborhoods = response.context["neighborhoods"]
-        self.assertEqual(neighborhoods.count(), 1)
+        json_neighborhoods = response.context["neighborhoods"]
+        neighborhoods = json.loads(json_neighborhoods)
+        tp = neighborhoods[0]["fields"]
+        neighborhood = Neighborhood.objects.all()
+        p = neighborhood[0].name
+        self.assertEqual(p, tp["name"])
```
Comment (vchrombie) [inline tests/test_views.py:696]: Please revert this change. The aim of the test is to check the the neighborhoods view and the count is just fine. I would suggest we keep the tests similar.

**Review by vchrombie — CHANGES_REQUESTED** [review]
I guess this PR needs some re-work and discussion.

**Comment by moringspeaker** [comment]
I don't think it's a Mapbox problem cause I tested it, it's because in this page we try to fetch a Bs icon everytime, but for the backend part, I admit I just write them for better mathing my frontend JS script and there's actually no improvement

**Comment by vchrombie** [comment]
> I don't think it's a Mapbox problem cause I tested it, it's because in this page we try to fetch a Bs icon everytime, but for the backend part, I admit I just write them for better mathing my frontend JS script and there's actually no improvement

I wasn't really convinced until you showed us in the meeting. But, I need to appreciate your work @moringspeaker. Thanks for all your experimentation.

Please let me know if you need help with the PR changes.

**Comment by vchrombie** [comment]
Rework was done in #121, this PR will be automatically closed when the other one is merged

**Pull request closed without merging**